packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1096,6 +1096,20 @@ This section contains some general information about claims, as well as detailed
 * [ID Token](#id-token)
 * [Refresh Token](#refresh-token)
 
+### Token lifetime
+
+When you are using the [Okta Authorization Server](/docs/concepts/auth-servers/#org-authorization-server), the lifetime of the JWT tokens is hard-coded to the following values:
+
+* **ID token:** 60 minutes
+* **Access token:** 60 minutes
+* **Refresh token:** 90 days
+
+When you are using a [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server), you can configure the lifetime of the JWT tokens:
+
+* **Access tokens:** At least five minutes, no more than 24 hours (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)).
+* **Refresh tokens:** The minimum active token lifetime, which is five minutes. The idle time window is at least 10 minutes, with a maximum of five years (configurable using an [Access Policy](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-create-access-policies)).
```

Comment:
The minimum lifetime is the same as the access token lifetime. `, which is five minutes` can be removed, if the admin set the access token as 10 minutes, then the refresh token minimum lifetime is 10 minutes.